students/DennisLee/lesson04/mailroom.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+
+# Initial donor list and the amounts they have donated
+donor_history = {
+            'Red Herring': [65820.5, 31126.37, 15000],
+            'Papa Smurf': [210.64, 1000],
+            'Pat Panda': [55324.4],
+            'Karl-Heinz Berthold': [3545.2, 10579.31],
+            'Mama Murphy': [156316.99, 8500.3, 12054.33],
+            'Daphne Dastardly': [82]
+        }
+
+def manage_donors():
+    """
+    Display the menu of choices for donor management.
+
+    :return:  None.
+    """
+    # create a dictionary of menu items, menu text, and menu caller functions
+    choices = {'1': ("Send a thank you", send_thank_you), 
+               '2': ("Create a report", create_a_report),
+               '3': ("Send letters to everyone", send_all_letters),
+               '4': ("Quit", exit_screen)}
+    
+    response = ''
+    while response != '4':  # Show menu forever until user exits
```

Comment:
I think this could be slightly simplified by using `while True` for this loop as well and placing `response` assignment inside the loop, so that you only need to assign once to an empty string. But you'd also need an `exit` function that actually exits the program